File: WPF客户端/AnnHome/AnnHome/Entity/Images.cs
namespace AnnHome.Entity
{
    internal class Images
    {
        private Img full;
        private Img thumbnail;
        private Img medium;
        private Img large;

        public Img Full
        {
            get
            {
                return full;
            }

            set
            {
                full = value;
            }
        }

        public Img Thumbnail
        {
            get
            {
                return thumbnail;
            }

            set
            {
                thumbnail = value;
            }
        }

        public Img Medium
        {
            get
            {
                return medium;
            }

            set
            {
                medium = value;
            }
        }

        public Img Large
        {
            get
            {
                return large;
            }

            set
            {
                large = value;
            }
        }
    }
}

File: Android客户端/AnnHome/app/src/main/java/entity/Images.java
package entity;

import java.io.Serializable;

/**
 * Created by mrpan on 15/11/17.
 */
public class Images extends BaseEntity implements Serializable {
    private static final long serialVersionUID = 1L;
    private Img full;
    private Img thumbnail;

    public Img getMedium() {
        return medium;
    }

    public void setMedium(Img medium) {
        this.medium = medium;
    }

    public Img getLarge() {
        return large;
    }

    public void setLarge(Img large) {
        this.large = large;
    }

    public Img getThumbnail() {
        return thumbnail;
    }

    public void setThumbnail(Img thumbnail) {
        this.thumbnail = thumbnail;
    }

    public Img getFull() {
        return full;
    }

    public void setFull(Img full) {
        this.full = full;
    }

    private Img medium;
    private Img large;
}
